sgl-router/src/core/workflow/steps/worker_registration.rs

```diff
@@ -105,6 +106,8 @@ async fn get_dp_info(url: &str, api_key: Option<&str>) -> Result<DpInfo, String>
 
     let model_id = info
         .model_id
+        .filter(|s| !s.is_empty())
+        .or(info.served_model_name.filter(|s| !s.is_empty()))
```

Comment:
I think served_model_name should have higher priority than model_id here? 
Although it doesn't matter much as it seems that this function is called only for debug logging purpose right now.